config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```diff
@@ -105,6 +105,8 @@ periodics:
           value: "default"
         - name: PROMETHEUS_APISERVER_SCRAPE_PORT
           value: "6443"
+        - name: NODE_PRELOAD_IMAGES
```

Comment:
just checking we want to keep this test pinned to your dra-baseline branch?

Reply:
no we dont need it pinned anymore, since we can allow that branch to merge.

master will pass with this and qps/burst infra change